File: src/main/java/com/xu/music/player/hander/DataBaseError.java
package com.xu.music.player.hander;

/**
 * 数据库异常
 *
 * @author hyacinth
 * @since 2024年6月4日19点07分
 * @version swt-java/v1.0.0
 */
public class DataBaseError extends RuntimeException {

    public DataBaseError(String msg) {
        super(msg);
    }

    public DataBaseError(Throwable cause) {
        super(cause);
    }

    public DataBaseError(String msg, Throwable cause) {
        super(msg, cause);
    }

}

File: src/main/kotlin/com/xu/music/player/hander/DataBaseError.kt
package com.xu.music.player.hander

/**
 * 数据库异常
 *
 * @date 2024年6月10日15点30分
 * @since V1.0.0.0
 */
class DataBaseError : RuntimeException {

    constructor(msg: String?) : super(msg)

    constructor(cause: Throwable?) : super(cause)

    constructor(msg: String?, cause: Throwable?) : super(msg, cause)

}
